### DISCUSSION TITLE
Why are Chinese characters in comments rendered to be some random symbols in presenter notes?

### DISCUSSION THREAD

**Discussion by guzy0324** [marp-team/marp#358] [Q&A] [answered]
I am using Marp for VS Code v1.5.2. And I find that the comment

```markdown
<!--
其中一个好的分解要满足以下条件：

- 两个子问题
- 没有子问题包含多跳问题的全部单词
- 没有子问题比多跳问题的长度长
-->
```

is rendered as

![image](https://user-images.githubusercontent.com/53924636/195235786-da8378f9-78cd-4284-826d-821263e93fa2.png)

Is it a bug or misconfiguration?

**Comment by yhatt** [comment, marked as answer]
It's a bug of the browser you're using. Chrome and Chromium-flavored browsers do not render notes with CJK characters in PDF correctly. It causes not only to PDF exported by Marp, but also every PDFs.

Recommend to use another PDF reader application, such as Adobe Acrobat Reader, Firefox's PDF viewer, and macOS Preview app.

**Reply by guzy0324** [reply]
Thanks.
